.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: haxe
+
+haxe:
+  - "3.2.0"
+  - development
+  
+env: 
+  - TARGET=swf
+  - TARGET=as3
+  - TARGET=js
+  - TARGET=neko
+  - TARGET=cpp
+  - TARGET=cs
+  - TARGET=java
+  - TARGET=python
+
+install:
+  - sudo apt-get install mono-devel mono-mcs -qq
```

Comment:
@Gama11 Let's use the [apt addon](http://docs.travis-ci.com/user/apt/) instead. It will allow the builds to start much faster with the container-based infrastructure.
e.g.
```yml
addons:
  apt:
    packages:
      - mono-devel
      - mono-mcs
```